src/main/java/kr/heesu/lotto/domain/Lotto.java

```diff
@@ -28,17 +28,16 @@ private static void validationCheck(List<LottoNumber> numbers) {
         }
     }
 
-    private boolean contain(LottoNumber number) {
+    public boolean contain(LottoNumber number) {
         return this.numbers.contains(number);
```

Comment:
```suggestion
        return numbers.contains(number);
```
굳이 this를 사용하지 않아도 될 것 같아요!

Reply:
불필요한 this 사용 제거하였습니다!